packages/node-cli/src/commands/ManageAccountCommandGenerator.js

```diff
@@ -0,0 +1,263 @@
+      }
+
+      return this._extractAnswers(answers);
+   }
+
+   async _selectAuthID(authIDList, prompt) {
+      var authIDs = Object.entries(authIDList).sort();
+      if (authIDs.length <= 0) {
+         throw NodeTranslationService.getMessage(ERRORS.CREDENTIALS_EMPTY);
+      }
+      const choices = [];
+      authIDs.forEach((authIDArray) => {
+         const authID = authIDArray[0];
+         const accountCredential = authIDArray[1];
+         const accountCredentialString = this.outputFormatter.accountCredentialToString(authID, accountCredential);
+         choices.push({
+            name: accountCredentialString,
+            value: { authId: authID, accountInfo: accountCredential.accountInfo, domain: accountCredential.urls.app },
+         });
+      });
+      choices.push(new inquirer.Separator());
+      let answers = await prompt([
+         {
+            type: CommandUtils.INQUIRER_TYPES.LIST,
+            name: ANSWERS_NAMES.SELECTED_AUTH_ID,
+            message: NodeTranslationService.getMessage(MESSAGES.SELECT_CONFIGURED_AUTHID),
+            choices: choices,
+         },
+      ]);
+      return answers;
+   }
+
+   async _selectAction(prompt) {
+      let answer = await prompt({
+         type: CommandUtils.INQUIRER_TYPES.LIST,
+         name: ANSWERS_NAMES.ACTION,
+         message: NodeTranslationService.getMessage(QUESTIONS.SELECT_ACTION),
+         choices: [
+            {
+               name: NodeTranslationService.getMessage(QUESTIONS_CHOICES.ACTIONS.RENAME),
+               value: ACTION.RENAME,
+            },
+            {
+               name: NodeTranslationService.getMessage(QUESTIONS_CHOICES.ACTIONS.REMOVE),
+               value: ACTION.REMOVE,
+            },
+            // {
+            //    name: NodeTranslationService.getMessage(QUESTIONS_CHOICES.ACTIONS.REVOKE),
+            //    value: ACTION.REVOKE,
+            // },
+            {
+               name: NodeTranslationService.getMessage(QUESTIONS_CHOICES.ACTIONS.EXIT),
+               value: ACTION.EXIT,
+            },
+         ],
+      });
+
+      if (answer[ANSWERS_NAMES.ACTION] == ACTION.EXIT) {
+         throw NodeTranslationService.getMessage(MESSAGES.CANCEL);
+      }
+
+      return answer[ANSWERS_NAMES.ACTION];
+   }
+
+   async _introduceNewName(prompt, authIDList, originalAuthId) {
+      let answer = await prompt({
+         type: CommandUtils.INQUIRER_TYPES.INPUT,
+         name: ANSWERS_NAMES.RENAMETO,
+         message: NodeTranslationService.getMessage(QUESTIONS.NEW_NAME),
+         filter: (fieldValue) => fieldValue.trim(),
+         validate: (fieldValue) =>
+            showValidationResults(
+               fieldValue,
+               validateFieldIsNotEmpty,
+               validateFieldHasNoSpaces,
+               (fieldValue) => validateSameAuthID(fieldValue, originalAuthId),
+               (fieldValue) => validateAuthIDNotInList(fieldValue, Object.keys(authIDList)),
```

Comment:
As the originalAuthId is already in the authIDList maybe we can skip that validateSameAuthID ?

Reply:
It sends a different error message so It's worth it to have it separated.